### PR TITLE
add nft actions tab and detail transfered token

### LINKED ISSUE #1537: NFT updates
[] add NFT token to action detail page.
https://iotexscan.io/action/8b536ff5eb4f751d402585824bdc4345ba2d7cd583743d1d54b7698fd24938e2
image

[] add NFT actions to account detail page
such as 
https://iotexscan.io/address/io1xpvzahnl4h46f9ea6u03ec2hkusrzu020th8xx
image

### PR DESCRIPTION
**Pull request by hunglmtb**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1537

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by hunglmtb** [comment]
**action detail page**
image




**address detail page**
image

File: src/shared/pages/address-details-page.tsx [hunk 1]
```diff
@@ -152,6 +153,12 @@ const AddressDetailsPage: React.FC<RouteComponentProps<{ address: string }>> = (
                 >
                   <XRC20ActionTable address={address} />
                 </Tabs.TabPane>
+                <Tabs.TabPane
+                  tab={t("common.xrc721Transactions")}
+                  key="xrcxrc721_transactions"
```
Comment (guo) [inline src/shared/pages/address-details-page.tsx:158]: why two xrcxrc?

File: translations/en.yaml [hunk 2]
```diff
@@ -168,7 +168,7 @@ common.show_less: Show Less
 common.show_more: Show More
 common.transactions: Total Actions
 common.xrc20Transactions: XRC20 Actions
-common.xrc721Transactions: XRC20 Actions
+common.xrc721Transactions: NFT Actions
```
Comment (guo) [inline translations/en.yaml:171]: pls check other files and remove them if they are not translated well.

Comment (hunglmtb) [reply translations/en.yaml:171]: it's only available in `translations/en.yaml`

**Comment by liaojikunwork** [comment]
Screen Shot 2020-08-12 at 1 44 39 PM

On wide screens, can we show the full address? ( I updated the style of the first row )


Screen Shot 2020-08-12 at 1 49 06 PM
On smaller screens, can we improve the UI (alignment)?

**Pull request merged**